contracts/contracts/vault/VaultStorage.sol

```diff
@@ -73,7 +72,7 @@ contract VaultStorage is Initializable, Governable {
     // Redemption fee in basis points
     uint256 public redeemFeeBps;
     // Buffer of assets to keep in Vault to handle (most) withdrawals
-    uint256 public vaultBuffer;
+    uint256 public vaultBuffer; // No longer used
```

Comment:
Elsewhere we've used `_deprecated_vaultBuffer` for these, might be nice to make it non-public too

Reply:
Done.